--- 2012-03-20/filtered.scan	2014-12-19 11:23:41.274314775 -0800
+++ 2012-07-05/filtered.scan	2014-12-19 11:23:41.269314585 -0800
@@ -1,5 +1,5 @@
 
-Starting Nmap 5.50 ( http://nmap.org ) at 2012-03-20 04:37
+Starting Nmap 5.50 ( http://nmap.org ) at 2012-07-05 18:27
 Nmap scan report for 175.45.176.0
 
 Nmap scan report for 175.45.176.1
@@ -31,6 +31,13 @@
 Nmap scan report for 175.45.176.7
 
 Nmap scan report for 175.45.176.8
+Not shown: 65534 filtered ports
+PORT   STATE SERVICE VERSION
+53/tcp open  domain  ISC BIND 9.8.1
+Warning: OSScan results may be unreliable because we could not find at least 1 open and 1 closed port
+Device type: general purpose
+Running: Linux 2.6.X
+OS details: Linux 2.6.9 - 2.6.18
 
 Nmap scan report for 175.45.176.9
 Not shown: 65534 filtered ports
@@ -46,21 +53,25 @@
 PORT   STATE SERVICE VERSION
 25/tcp open  smtp    Cisco PIX sanitized smtpd
 Warning: OSScan results may be unreliable because we could not find at least 1 open and 1 closed port
-Device type: phone|firewall|general purpose
-Running (JUST GUESSING): Nokia Symbian OS (88%), Juniper embedded (88%), FreeBSD 6.X (87%), OpenBSD 4.X (86%)
-Aggressive OS guesses: Nokia N81 mobile phone (Symbian OS) (88%), Juniper Networks SSG 20 firewall (88%), FreeBSD 6.2-RELEASE (87%), FreeBSD 6.3-RELEASE (87%), OpenBSD 4.0 (86%), OpenBSD 4.3 (86%)
+Device type: general purpose|phone|firewall
+Running (JUST GUESSING): FreeBSD 6.X (89%), Nokia Symbian OS (87%), OpenBSD 4.X (87%), Juniper embedded (86%)
+Aggressive OS guesses: FreeBSD 6.3-RELEASE (89%), Nokia N81 mobile phone (Symbian OS) (87%), OpenBSD 4.0 (87%), Juniper Networks SSG 20 firewall (86%)
 No exact OS matches for host (test conditions non-ideal).
 Service Info: Device: firewall
 
 Nmap scan report for 175.45.176.11
 
 Nmap scan report for 175.45.176.12
-Not shown: 65481 filtered ports, 50 closed ports
+Not shown: 65481 filtered ports, 46 closed ports
 PORT      STATE SERVICE VERSION
 21/tcp    open  ftp     vsftpd
 7200/tcp  open  fodms?
+51005/tcp open  unknown
+51020/tcp open  unknown
 51022/tcp open  unknown
-51026/tcp open  unknown
+51039/tcp open  unknown
+51048/tcp open  unknown
+51053/tcp open  unknown
 Device type: general purpose
 Running: Linux 2.6.X
 OS details: Linux 2.6.18
@@ -191,86 +202,82 @@
 
 Nmap scan report for 175.45.176.64
 
-Nmap scan report for mailhost.silibank.net.kp (175.45.176.65)
-Not shown: 65532 filtered ports
+Nmap scan report for mail.silibank.net.kp (175.45.176.65)
+Not shown: 65533 filtered ports
 PORT      STATE SERVICE VERSION
 25/tcp    open  smtp    Cisco PIX sanitized smtpd
 60000/tcp open  unknown
-62037/tcp open  unknown
-2 services unrecognized despite returning data. If you know the service/version, please submit the following fingerprints at http://www.insecure.org/cgi-bin/servicefp-submit.cgi :
-==============NEXT SERVICE FINGERPRINT (SUBMIT INDIVIDUALLY)==============
-SF-Port60000-TCP:V=5.50%I=7%D=3/20%Time=4F68CA73%P=x86_64-redhat-linux-gnu
-SF:%r(NULL,C7,"\$CONNECT\r\nPROTOCOL:1\.0\.0\.1\r\nSTMVERSION:1\.0\.0\.7\r
-SF:\nSIZE:139\r\n\r\n0\x81\x88\x02\x81\x80\|c\xf1d\xec\xf0\xe5\"\t\xd4\x19
-SF:\xaec\"\x8f\x10\xd2\xc2\xc4\x17\x8br\x15\xd7\x14z\xc6'\xd3\x20\xa8\x0c\
-SF:xa87\xe2\x94f\x9c\xe4X\xd7\n\xeb\xcc\xb5\x0e\xec\x03u\xd6\xbcM\(CJ\xba<
-SF:SS\x19\xa9\x1e\xbf\xc1\x86\xbf@\xa5@;\x87\xe3\x9c\xc1\x8d\xec@\x0e\xb3\
-SF:x1cU\x7f\x80\x15}8,\xe3\xcfm\x8aR\xd7\xca\x91\x1d\x9e\(\xb1\x99\x829\x9
-SF:6\xb6\xe6j\xef\xa5\|b\[\xae\xc9%\x9d\xd1~\xb5A\x81@~\xfc\xf3\x7fO\x14\x
-SF:c1\x02\x03\x01\0\x01")%r(ibm-db2,C7,"\$CONNECT\r\nPROTOCOL:1\.0\.0\.1\r
-SF:\nSTMVERSION:1\.0\.0\.7\r\nSIZE:139\r\n\r\n0\x81\x88\x02\x81\x80\|c\xf1
-SF:d\xec\xf0\xe5\"\t\xd4\x19\xaec\"\x8f\x10\xd2\xc2\xc4\x17\x8br\x15\xd7\x
-SF:14z\xc6'\xd3\x20\xa8\x0c\xa87\xe2\x94f\x9c\xe4X\xd7\n\xeb\xcc\xb5\x0e\x
-SF:ec\x03u\xd6\xbcM\(CJ\xba<SS\x19\xa9\x1e\xbf\xc1\x86\xbf@\xa5@;\x87\xe3\
-SF:x9c\xc1\x8d\xec@\x0e\xb3\x1cU\x7f\x80\x15}8,\xe3\xcfm\x8aR\xd7\xca\x91\
-SF:x1d\x9e\(\xb1\x99\x829\x96\xb6\xe6j\xef\xa5\|b\[\xae\xc9%\x9d\xd1~\xb5A
-SF:\x81@~\xfc\xf3\x7fO\x14\xc1\x02\x03\x01\0\x01")%r(drda,C7,"\$CONNECT\r\
-SF:nPROTOCOL:1\.0\.0\.1\r\nSTMVERSION:1\.0\.0\.7\r\nSIZE:139\r\n\r\n0\x81\
-SF:x88\x02\x81\x80\|c\xf1d\xec\xf0\xe5\"\t\xd4\x19\xaec\"\x8f\x10\xd2\xc2\
-SF:xc4\x17\x8br\x15\xd7\x14z\xc6'\xd3\x20\xa8\x0c\xa87\xe2\x94f\x9c\xe4X\x
-SF:d7\n\xeb\xcc\xb5\x0e\xec\x03u\xd6\xbcM\(CJ\xba<SS\x19\xa9\x1e\xbf\xc1\x
-SF:86\xbf@\xa5@;\x87\xe3\x9c\xc1\x8d\xec@\x0e\xb3\x1cU\x7f\x80\x15}8,\xe3\
-SF:xcfm\x8aR\xd7\xca\x91\x1d\x9e\(\xb1\x99\x829\x96\xb6\xe6j\xef\xa5\|b\[\
-SF:xae\xc9%\x9d\xd1~\xb5A\x81@~\xfc\xf3\x7fO\x14\xc1\x02\x03\x01\0\x01")%r
-SF:(GenericLines,C7,"\$CONNECT\r\nPROTOCOL:1\.0\.0\.1\r\nSTMVERSION:1\.0\.
-SF:0\.7\r\nSIZE:139\r\n\r\n0\x81\x88\x02\x81\x80\|c\xf1d\xec\xf0\xe5\"\t\x
-SF:d4\x19\xaec\"\x8f\x10\xd2\xc2\xc4\x17\x8br\x15\xd7\x14z\xc6'\xd3\x20\xa
-SF:8\x0c\xa87\xe2\x94f\x9c\xe4X\xd7\n\xeb\xcc\xb5\x0e\xec\x03u\xd6\xbcM\(C
-SF:J\xba<SS\x19\xa9\x1e\xbf\xc1\x86\xbf@\xa5@;\x87\xe3\x9c\xc1\x8d\xec@\x0
-SF:e\xb3\x1cU\x7f\x80\x15}8,\xe3\xcfm\x8aR\xd7\xca\x91\x1d\x9e\(\xb1\x99\x
-SF:829\x96\xb6\xe6j\xef\xa5\|b\[\xae\xc9%\x9d\xd1~\xb5A\x81@~\xfc\xf3\x7fO
-SF:\x14\xc1\x02\x03\x01\0\x01");
-==============NEXT SERVICE FINGERPRINT (SUBMIT INDIVIDUALLY)==============
-SF-Port62037-TCP:V=5.50%I=7%D=3/20%Time=4F68CA73%P=x86_64-redhat-linux-gnu
-SF:%r(NULL,87,"\$CONNECT\r\nPROTOCOL:1\.0\r\nCLIENT:1\.0\.0\.4\r\nSIZE:84\
-SF:r\n\r\n\x06\x02\0\0\0\xa4\0\0RSA1\0\x02\0\0\x01\0\x01\0\x81\xbc\x18\x12
-SF:\\\x19\x9fRz\xa32%\"\x98p\xcdb0T\x06\xda\xcdq\xfd%z\xc8\x0b\x98\xbfKj\x
-SF:9ej\xc1\xe8g\x19Nh\x04\x08~T\xc8\x12\xa1\xd5y\xaf\+\xca\xbb\xdb#q\\\]4\
-SF:.\x17q\x94\xc9")%r(GenericLines,87,"\$CONNECT\r\nPROTOCOL:1\.0\r\nCLIEN
-SF:T:1\.0\.0\.4\r\nSIZE:84\r\n\r\n\x06\x02\0\0\0\xa4\0\0RSA1\0\x02\0\0\x01
-SF:\0\x01\0\x81\xbc\x18\x12\\\x19\x9fRz\xa32%\"\x98p\xcdb0T\x06\xda\xcdq\x
-SF:fd%z\xc8\x0b\x98\xbfKj\x9ej\xc1\xe8g\x19Nh\x04\x08~T\xc8\x12\xa1\xd5y\x
-SF:af\+\xca\xbb\xdb#q\\\]4\.\x17q\x94\xc9")%r(GetRequest,87,"\$CONNECT\r\n
-SF:PROTOCOL:1\.0\r\nCLIENT:1\.0\.0\.4\r\nSIZE:84\r\n\r\n\x06\x02\0\0\0\xa4
-SF:\0\0RSA1\0\x02\0\0\x01\0\x01\0\x81\xbc\x18\x12\\\x19\x9fRz\xa32%\"\x98p
-SF:\xcdb0T\x06\xda\xcdq\xfd%z\xc8\x0b\x98\xbfKj\x9ej\xc1\xe8g\x19Nh\x04\x0
-SF:8~T\xc8\x12\xa1\xd5y\xaf\+\xca\xbb\xdb#q\\\]4\.\x17q\x94\xc9")%r(HTTPOp
-SF:tions,87,"\$CONNECT\r\nPROTOCOL:1\.0\r\nCLIENT:1\.0\.0\.4\r\nSIZE:84\r\
-SF:n\r\n\x06\x02\0\0\0\xa4\0\0RSA1\0\x02\0\0\x01\0\x01\0\x81\xbc\x18\x12\\
-SF:\x19\x9fRz\xa32%\"\x98p\xcdb0T\x06\xda\xcdq\xfd%z\xc8\x0b\x98\xbfKj\x9e
-SF:j\xc1\xe8g\x19Nh\x04\x08~T\xc8\x12\xa1\xd5y\xaf\+\xca\xbb\xdb#q\\\]4\.\
-SF:x17q\x94\xc9")%r(RTSPRequest,87,"\$CONNECT\r\nPROTOCOL:1\.0\r\nCLIENT:1
-SF:\.0\.0\.4\r\nSIZE:84\r\n\r\n\x06\x02\0\0\0\xa4\0\0RSA1\0\x02\0\0\x01\0\
-SF:x01\0\x81\xbc\x18\x12\\\x19\x9fRz\xa32%\"\x98p\xcdb0T\x06\xda\xcdq\xfd%
-SF:z\xc8\x0b\x98\xbfKj\x9ej\xc1\xe8g\x19Nh\x04\x08~T\xc8\x12\xa1\xd5y\xaf\
-SF:+\xca\xbb\xdb#q\\\]4\.\x17q\x94\xc9")%r(RPCCheck,87,"\$CONNECT\r\nPROTO
-SF:COL:1\.0\r\nCLIENT:1\.0\.0\.4\r\nSIZE:84\r\n\r\n\x06\x02\0\0\0\xa4\0\0R
-SF:SA1\0\x02\0\0\x01\0\x01\0\x81\xbc\x18\x12\\\x19\x9fRz\xa32%\"\x98p\xcdb
-SF:0T\x06\xda\xcdq\xfd%z\xc8\x0b\x98\xbfKj\x9ej\xc1\xe8g\x19Nh\x04\x08~T\x
-SF:c8\x12\xa1\xd5y\xaf\+\xca\xbb\xdb#q\\\]4\.\x17q\x94\xc9")%r(DNSVersionB
-SF:indReq,87,"\$CONNECT\r\nPROTOCOL:1\.0\r\nCLIENT:1\.0\.0\.4\r\nSIZE:84\r
-SF:\n\r\n\x06\x02\0\0\0\xa4\0\0RSA1\0\x02\0\0\x01\0\x01\0\x81\xbc\x18\x12\
-SF:\\x19\x9fRz\xa32%\"\x98p\xcdb0T\x06\xda\xcdq\xfd%z\xc8\x0b\x98\xbfKj\x9
-SF:ej\xc1\xe8g\x19Nh\x04\x08~T\xc8\x12\xa1\xd5y\xaf\+\xca\xbb\xdb#q\\\]4\.
-SF:\x17q\x94\xc9");
+1 service unrecognized despite returning data. If you know the service/version, please submit the following fingerprint at http://www.insecure.org/cgi-bin/servicefp-submit.cgi :
+SF-Port60000-TCP:V=5.50%I=7%D=7/6%Time=4FF699E5%P=x86_64-redhat-linux-gnu%
+SF:r(NULL,C7,"\$CONNECT\r\nPROTOCOL:1\.0\.0\.1\r\nSTMVERSION:1\.0\.0\.7\r\
+SF:nSIZE:139\r\n\r\n0\x81\x88\x02\x81\x80D\x8f\x19z\xf8\xa2l\x02r/\xab_,\x
+SF:a52b\)\xa8#\)\xbbIZ\xce\x1f\xb0GOkK\x81<1\x08\x92L\?\xb95\|\x07\xbfVG\x
+SF:91l\xbb\x7f\xd2\xad\x9f\x95\xd6\xccxS\x14y6\xd8\xee\xd1\xf0\x87\xa1k\x8
+SF:6\x9ew\xaeQ\xd5x\x0c\x83R\x17\xce\x20\xc27\xb6W\xdc\xd1A\xf8OO3Y\x84\xd
+SF:dd\xcd\xe3\.\xf0\x13l4\xaa\xc0\xba\x83\xed\*D\xbc\x17\x07\0i6o\xb9\xa3\
+SF:.i\xbf\x85\xb7\$\x06\xa3\x11\xdb/\x02\x03\x01\0\x01")%r(ibm-db2,C7,"\$C
+SF:ONNECT\r\nPROTOCOL:1\.0\.0\.1\r\nSTMVERSION:1\.0\.0\.7\r\nSIZE:139\r\n\
+SF:r\n0\x81\x88\x02\x81\x80D\x8f\x19z\xf8\xa2l\x02r/\xab_,\xa52b\)\xa8#\)\
+SF:xbbIZ\xce\x1f\xb0GOkK\x81<1\x08\x92L\?\xb95\|\x07\xbfVG\x91l\xbb\x7f\xd
+SF:2\xad\x9f\x95\xd6\xccxS\x14y6\xd8\xee\xd1\xf0\x87\xa1k\x86\x9ew\xaeQ\xd
+SF:5x\x0c\x83R\x17\xce\x20\xc27\xb6W\xdc\xd1A\xf8OO3Y\x84\xddd\xcd\xe3\.\x
+SF:f0\x13l4\xaa\xc0\xba\x83\xed\*D\xbc\x17\x07\0i6o\xb9\xa3\.i\xbf\x85\xb7
+SF:\$\x06\xa3\x11\xdb/\x02\x03\x01\0\x01")%r(drda,C7,"\$CONNECT\r\nPROTOCO
+SF:L:1\.0\.0\.1\r\nSTMVERSION:1\.0\.0\.7\r\nSIZE:139\r\n\r\n0\x81\x88\x02\
+SF:x81\x80D\x8f\x19z\xf8\xa2l\x02r/\xab_,\xa52b\)\xa8#\)\xbbIZ\xce\x1f\xb0
+SF:GOkK\x81<1\x08\x92L\?\xb95\|\x07\xbfVG\x91l\xbb\x7f\xd2\xad\x9f\x95\xd6
+SF:\xccxS\x14y6\xd8\xee\xd1\xf0\x87\xa1k\x86\x9ew\xaeQ\xd5x\x0c\x83R\x17\x
+SF:ce\x20\xc27\xb6W\xdc\xd1A\xf8OO3Y\x84\xddd\xcd\xe3\.\xf0\x13l4\xaa\xc0\
+SF:xba\x83\xed\*D\xbc\x17\x07\0i6o\xb9\xa3\.i\xbf\x85\xb7\$\x06\xa3\x11\xd
+SF:b/\x02\x03\x01\0\x01")%r(GenericLines,C7,"\$CONNECT\r\nPROTOCOL:1\.0\.0
+SF:\.1\r\nSTMVERSION:1\.0\.0\.7\r\nSIZE:139\r\n\r\n0\x81\x88\x02\x81\x80D\
+SF:x8f\x19z\xf8\xa2l\x02r/\xab_,\xa52b\)\xa8#\)\xbbIZ\xce\x1f\xb0GOkK\x81<
+SF:1\x08\x92L\?\xb95\|\x07\xbfVG\x91l\xbb\x7f\xd2\xad\x9f\x95\xd6\xccxS\x1
+SF:4y6\xd8\xee\xd1\xf0\x87\xa1k\x86\x9ew\xaeQ\xd5x\x0c\x83R\x17\xce\x20\xc
+SF:27\xb6W\xdc\xd1A\xf8OO3Y\x84\xddd\xcd\xe3\.\xf0\x13l4\xaa\xc0\xba\x83\x
+SF:ed\*D\xbc\x17\x07\0i6o\xb9\xa3\.i\xbf\x85\xb7\$\x06\xa3\x11\xdb/\x02\x0
+SF:3\x01\0\x01")%r(GetRequest,C7,"\$CONNECT\r\nPROTOCOL:1\.0\.0\.1\r\nSTMV
+SF:ERSION:1\.0\.0\.7\r\nSIZE:139\r\n\r\n0\x81\x88\x02\x81\x80D\x8f\x19z\xf
+SF:8\xa2l\x02r/\xab_,\xa52b\)\xa8#\)\xbbIZ\xce\x1f\xb0GOkK\x81<1\x08\x92L\
+SF:?\xb95\|\x07\xbfVG\x91l\xbb\x7f\xd2\xad\x9f\x95\xd6\xccxS\x14y6\xd8\xee
+SF:\xd1\xf0\x87\xa1k\x86\x9ew\xaeQ\xd5x\x0c\x83R\x17\xce\x20\xc27\xb6W\xdc
+SF:\xd1A\xf8OO3Y\x84\xddd\xcd\xe3\.\xf0\x13l4\xaa\xc0\xba\x83\xed\*D\xbc\x
+SF:17\x07\0i6o\xb9\xa3\.i\xbf\x85\xb7\$\x06\xa3\x11\xdb/\x02\x03\x01\0\x01
+SF:");
 Warning: OSScan results may be unreliable because we could not find at least 1 open and 1 closed port
 Device type: general purpose|WAP
 Running (JUST GUESSING): Microsoft Windows 2003 (91%), Apple embedded (86%)
-Aggressive OS guesses: Microsoft Windows Server 2003 SP1 (91%), Microsoft Windows Server 2003 SP1 or SP2 (89%), Microsoft Windows Server 2003 SP2 (89%), Apple AirPort Extreme WAP v7.3.2 (86%)
+Aggressive OS guesses: Microsoft Windows Server 2003 SP1 (91%), Microsoft Windows Server 2003 SP1 or SP2 (87%), Microsoft Windows Server 2003 SP2 (87%), Apple AirPort Extreme WAP v7.3.2 (86%)
 No exact OS matches for host (test conditions non-ideal).
 Service Info: Device: firewall
 
 Nmap scan report for 175.45.176.66
+Not shown: 65515 closed ports
+PORT     STATE    SERVICE        VERSION
+80/tcp   open     http           Microsoft IIS httpd 6.0
+135/tcp  filtered msrpc
+136/tcp  filtered profile
+137/tcp  filtered netbios-ns
+138/tcp  filtered netbios-dgm
+139/tcp  filtered netbios-ssn
+443/tcp  open     ssl/http       Microsoft IIS httpd 6.0
+445/tcp  filtered microsoft-ds
+593/tcp  filtered http-rpc-epmap
+1025/tcp open     msrpc          Microsoft Windows RPC
+1026/tcp open     msrpc          Microsoft Windows RPC
+1027/tcp open     msrpc          Microsoft Windows RPC
+1036/tcp open     msrpc          Microsoft Windows RPC
+1433/tcp open     ms-sql-s       Microsoft SQL Server 2000 8.00.311; RTMa
+1801/tcp open     msmq?
+2103/tcp open     msrpc          Microsoft Windows RPC
+2105/tcp open     msrpc          Microsoft Windows RPC
+2107/tcp open     msrpc          Microsoft Windows RPC
+3389/tcp open     microsoft-rdp  Microsoft Terminal Service
+4444/tcp filtered krb524
+Device type: general purpose
+Running: Microsoft Windows 2003
+OS details: Microsoft Windows Server 2003 SP1
+Service Info: OS: Windows
 
 Nmap scan report for naenara.com.kp (175.45.176.67)
 Not shown: 65526 filtered ports
@@ -419,20 +426,21 @@
 Nmap scan report for 175.45.176.128
 
 Nmap scan report for 175.45.176.129
-Not shown: 65527 closed ports
-PORT     STATE    SERVICE        VERSION
-135/tcp  filtered msrpc
-136/tcp  filtered profile
-137/tcp  filtered netbios-ns
-138/tcp  filtered netbios-dgm
-139/tcp  filtered netbios-ssn
-445/tcp  filtered microsoft-ds
-593/tcp  filtered http-rpc-epmap
-4444/tcp filtered krb524
+Not shown: 65526 closed ports
+PORT      STATE    SERVICE        VERSION
+135/tcp   filtered msrpc
+136/tcp   filtered profile
+137/tcp   filtered netbios-ns
+138/tcp   filtered netbios-dgm
+139/tcp   filtered netbios-ssn
+445/tcp   filtered microsoft-ds
+593/tcp   filtered http-rpc-epmap
+4444/tcp  filtered krb524
+52903/tcp filtered unknown
 Warning: OSScan results may be unreliable because we could not find at least 1 open and 1 closed port
 Device type: firewall|webcam|router|WAP
-Running: Cisco PIX OS 8.X, D-Link embedded, Linksys embedded, Planet embedded
-OS details: Cisco ASA 5540 firewall (PIX OS 8.0), Linksys BEFSR41 EtherFast router or D-Link DCS-6620G webcam, Linksys BEFSR41 EtherFast router, Linksys BEFSR81 router, Linksys BEFW11S4 WAP, Linksys WRK54G WAP, Planet WAP-1950 WAP
+Running: Cisco PIX OS 8.X, D-Link embedded, Linksys embedded
+OS details: Cisco ASA 5540 firewall (PIX OS 8.0), Linksys BEFSR41 EtherFast router or D-Link DCS-6620G webcam, Linksys BEFSR81 router, Linksys BEFW11S4 WAP
 
 Nmap scan report for 175.45.176.130
 
@@ -763,16 +771,6 @@
 Nmap scan report for 175.45.177.32
 
 Nmap scan report for 175.45.177.33
-Not shown: 65527 closed ports
-PORT     STATE    SERVICE        VERSION
-135/tcp  filtered msrpc
-136/tcp  filtered profile
-137/tcp  filtered netbios-ns
-138/tcp  filtered netbios-dgm
-139/tcp  filtered netbios-ssn
-445/tcp  filtered microsoft-ds
-593/tcp  filtered http-rpc-epmap
-4444/tcp filtered krb524
 
 Nmap scan report for 175.45.177.34
 
@@ -781,16 +779,6 @@
 Nmap scan report for 175.45.177.36
 
 Nmap scan report for 175.45.177.37
-Not shown: 65527 closed ports
-PORT     STATE    SERVICE        VERSION
-135/tcp  filtered msrpc
-136/tcp  filtered profile
-137/tcp  filtered netbios-ns
-138/tcp  filtered netbios-dgm
-139/tcp  filtered netbios-ssn
-445/tcp  filtered microsoft-ds
-593/tcp  filtered http-rpc-epmap
-4444/tcp filtered krb524
 
 Nmap scan report for 175.45.177.38
 
@@ -1100,34 +1088,12 @@
 
 Nmap scan report for 175.45.177.191
 
+Warning: 175.45.177.210 giving up on port because retransmission cap hit (6).
 Nmap scan report for 175.45.177.192
 
 Nmap scan report for 175.45.177.193
-Not shown: 65527 closed ports
-PORT     STATE    SERVICE        VERSION
-135/tcp  filtered msrpc
-136/tcp  filtered profile
-137/tcp  filtered netbios-ns
-138/tcp  filtered netbios-dgm
-139/tcp  filtered netbios-ssn
-445/tcp  filtered microsoft-ds
-593/tcp  filtered http-rpc-epmap
-4444/tcp filtered krb524
 
 Nmap scan report for 175.45.177.194
-Not shown: 65527 closed ports
-PORT     STATE    SERVICE        VERSION
-135/tcp  filtered msrpc
-136/tcp  filtered profile
-137/tcp  filtered netbios-ns
-138/tcp  filtered netbios-dgm
-139/tcp  filtered netbios-ssn
-445/tcp  filtered microsoft-ds
-593/tcp  filtered http-rpc-epmap
-4444/tcp filtered krb524
-Device type: switch|broadband router|router|WAP
-Running: Cisco CatOS, Cisco embedded, Cisco IOS 11.X|12.X|15.X
-Network Distance: 16 hops
 
 Nmap scan report for 175.45.177.195
 
@@ -1142,16 +1108,6 @@
 Nmap scan report for 175.45.177.200
 
 Nmap scan report for 175.45.177.201
-Not shown: 65527 closed ports
-PORT     STATE    SERVICE        VERSION
-135/tcp  filtered msrpc
-136/tcp  filtered profile
-137/tcp  filtered netbios-ns
-138/tcp  filtered netbios-dgm
-139/tcp  filtered netbios-ssn
-445/tcp  filtered microsoft-ds
-593/tcp  filtered http-rpc-epmap
-4444/tcp filtered krb524
 
 Nmap scan report for 175.45.177.202
 
@@ -1168,110 +1124,12 @@
 Nmap scan report for 175.45.177.208
 
 Nmap scan report for 175.45.177.209
-Not shown: 65527 closed ports
-PORT     STATE    SERVICE        VERSION
-135/tcp  filtered msrpc
-136/tcp  filtered profile
-137/tcp  filtered netbios-ns
-138/tcp  filtered netbios-dgm
-139/tcp  filtered netbios-ssn
-445/tcp  filtered microsoft-ds
-593/tcp  filtered http-rpc-epmap
-4444/tcp filtered krb524
 
 Nmap scan report for 175.45.177.210
-Not shown: 65521 closed ports
-PORT     STATE    SERVICE        VERSION
-22/tcp   open     ssh            (protocol 1.99)
-23/tcp   open     tcpwrapped
-135/tcp  filtered msrpc
-136/tcp  filtered profile
-137/tcp  filtered netbios-ns
-138/tcp  filtered netbios-dgm
-139/tcp  filtered netbios-ssn
-445/tcp  filtered microsoft-ds
-593/tcp  filtered http-rpc-epmap
-2049/tcp open     nfs?
-4321/tcp open     ssl/rwhois?
-4444/tcp filtered krb524
-6543/tcp filtered mythtv
-6545/tcp filtered unknown
-2 services unrecognized despite returning data. If you know the service/version, please submit the following fingerprints at http://www.insecure.org/cgi-bin/servicefp-submit.cgi :
-==============NEXT SERVICE FINGERPRINT (SUBMIT INDIVIDUALLY)==============
-SF-Port22-TCP:V=5.50%I=7%D=3/28%Time=4F737814%P=x86_64-redhat-linux-gnu%r(
-SF:NULL,B,"SSH-1\.99--\n");
-==============NEXT SERVICE FINGERPRINT (SUBMIT INDIVIDUALLY)==============
-SF-Port4321-TCP:V=5.50%T=SSL%I=7%D=3/28%Time=4F737849%P=x86_64-redhat-linu
-SF:x-gnu%r(GenericLines,9F,"HTTP/1\.1\x20400\x20\r\nDate:\x20SUN,\x2029\x2
-SF:0Mar\x202012\x2005:44:36\r\nConnection:\x20Keep-Alive\r\nServer:\x20HTT
-SF:P\x20Server\x201\.0\r\nContent-Length:\x200\r\nContent-Type:\x20text/ht
-SF:ml;\x20charset=gb2312\r\n\r\n")%r(GetRequest,1B4A,"HTTP/1\.1\x20200\x20
-SF:OK\r\nDate:\x20SUN,\x2029\x20Mar\x202012\x2005:44:37\r\nConnection:\x20
-SF:Keep-Alive\r\nServer:\x20HTTP\x20Server\x201\.0\r\nContent-Length:\x206
-SF:706\r\nContent-Type:\x20text/html;\x20charset=gb2312\r\nSet-Cookie:\x20
-SF:SESSIONID=OwAAAC6CGSRuVtgPeiCsjw==&OwAAAC6CGSRuVtgPeiCsjw==&Secoway&lan
-SF:gfrombrows=&copyright=2008-2011\r\n\r\n<html>\r\n\t<head>\r\n\t\t<!--\x
-SF:20HTTP\x201\.1\x20-->\r\n\x20\x20\x20\x20\x20\x20\x20\x20<meta\x20http-
-SF:equiv=\"Cache-Control\"\x20content=\"no-store\"/>\r\n\x20\x20\x20\x20\x
-SF:20\x20\x20\x20<!--\x20HTTP\x201\.0\x20-->\r\n\x20\x20\x20\x20\x20\x20\x
-SF:20\x20<meta\x20http-equiv=\"Pragma\"\x20content=\"no-cache\"/>\r\n\x20\
-SF:x20\x20\x20\x20\x20\x20\x20<!--\x20Prevents\x20caching\x20at\x20the\x20
-SF:Proxy\x20Server\x20-->\r\n\x20\x20\x20\x20\x20\x20\x20\x20<meta\x20http
-SF:-equiv=\"Expires\"\x20content=\"0\"/>\r\n\t\t<meta\x20http-equiv=\"Cont
-SF:ent-Type\"\x20content=\"text/html;\x20charset=UTF-8\"></meta>\r\n\t\t<t
-SF:itle></title>\r\n\t\t<link\x20rel=\"stylesheet\"\x20type=\"text/css\"\r
-SF:\n\t\t\thref=\"ext-lib/ext/resources/css/ext-all\.css\"\x20/>\r\n\t\t<l
-SF:ink\x20rel=\"stylesheet\"\x20type=\"text/css\"\x20href=\"sweb-lib/resou
-SF:rce/style/sweb-cmp\.css\"></link>\r\n\t\t<link\x20rel=\"stylesheet\"\x2
-SF:0type=\"text/css\"\x20href=")%r(HTTPOptions,1B4A,"HTTP/1\.1\x20200\x20O
-SF:K\r\nDate:\x20SUN,\x2029\x20Mar\x202012\x2005:44:39\r\nConnection:\x20K
-SF:eep-Alive\r\nServer:\x20HTTP\x20Server\x201\.0\r\nContent-Length:\x2067
-SF:06\r\nContent-Type:\x20text/html;\x20charset=gb2312\r\nSet-Cookie:\x20S
-SF:ESSIONID=PAAAAPaD6ljrTNBdIXBFoA==&PAAAAPaD6ljrTNBdIXBFoA==&Secoway&lang
-SF:frombrows=&copyright=2008-2011\r\n\r\n<html>\r\n\t<head>\r\n\t\t<!--\x2
-SF:0HTTP\x201\.1\x20-->\r\n\x20\x20\x20\x20\x20\x20\x20\x20<meta\x20http-e
-SF:quiv=\"Cache-Control\"\x20content=\"no-store\"/>\r\n\x20\x20\x20\x20\x2
-SF:0\x20\x20\x20<!--\x20HTTP\x201\.0\x20-->\r\n\x20\x20\x20\x20\x20\x20\x2
-SF:0\x20<meta\x20http-equiv=\"Pragma\"\x20content=\"no-cache\"/>\r\n\x20\x
-SF:20\x20\x20\x20\x20\x20\x20<!--\x20Prevents\x20caching\x20at\x20the\x20P
-SF:roxy\x20Server\x20-->\r\n\x20\x20\x20\x20\x20\x20\x20\x20<meta\x20http-
-SF:equiv=\"Expires\"\x20content=\"0\"/>\r\n\t\t<meta\x20http-equiv=\"Conte
-SF:nt-Type\"\x20content=\"text/html;\x20charset=UTF-8\"></meta>\r\n\t\t<ti
-SF:tle></title>\r\n\t\t<link\x20rel=\"stylesheet\"\x20type=\"text/css\"\r\
-SF:n\t\t\thref=\"ext-lib/ext/resources/css/ext-all\.css\"\x20/>\r\n\t\t<li
-SF:nk\x20rel=\"stylesheet\"\x20type=\"text/css\"\x20href=\"sweb-lib/resour
-SF:ce/style/sweb-cmp\.css\"></link>\r\n\t\t<link\x20rel=\"stylesheet\"\x20
-SF:type=\"text/css\"\x20href=");
-Device type: switch|firewall|broadband router|VoIP phone|WAP
-Running (JUST GUESSING): 3Com embedded (93%), ZyXEL ZyNOS 4.X|3.X (92%), Motorola embedded (89%), HP embedded (88%), Polycom embedded (88%), AirSpan embedded (85%), Siemens embedded (85%)
-Aggressive OS guesses: 3Com 5500-EI switch (93%), ZyXEL ZyWALL 5 firewall (92%), ZyXEL ZyWALL 2, 5, or 70 firewall (ZyNOS 4.04) (91%), Motorola SURFboard SB5100i cable modem (89%), 3Com SuperStack 3 Switch 4500 (88%), HP Brocade 1600 switch (88%), Polycom SoundPoint 501 IP phone (88%), ZyXEL ZyWALL 2 firewall or Prestige 660HW-61 ADSL router (ZyNOS 3.62) (87%), ZyXEL Prestige 660HW-61 ADSL router (ZyNOS 3.40) (87%), ZyXEL Prestige 660HW-D1 wireless ADSL router (86%)
-No exact OS matches for host (test conditions non-ideal).
 
 Nmap scan report for 175.45.177.211
-Not shown: 65526 closed ports
-PORT     STATE    SERVICE        VERSION
-80/tcp   filtered http
-135/tcp  filtered msrpc
-136/tcp  filtered profile
-137/tcp  filtered netbios-ns
-138/tcp  filtered netbios-dgm
-139/tcp  filtered netbios-ssn
-445/tcp  filtered microsoft-ds
-593/tcp  filtered http-rpc-epmap
-4444/tcp filtered krb524
-Warning: OSScan results may be unreliable because we could not find at least 1 open and 1 closed port
-Device type: broadband router|specialized
-Running: D-Link embedded, Motorola embedded
-OS details: D-Link DIR-100 DSL router or Motorola Canopy cluster management module 2.2
-Network Distance: 15 hops
 
 Nmap scan report for 175.45.177.212
-Not shown: 65534 filtered ports
-PORT    STATE SERVICE VERSION
-443/tcp open  https?
-Warning: OSScan results may be unreliable because we could not find at least 1 open and 1 closed port
-OS fingerprint not ideal because: Missing a closed TCP port so results incomplete
-No OS matches for host
 
 Nmap scan report for 175.45.177.213
 
@@ -1284,6 +1142,11 @@
 Nmap scan report for 175.45.177.217
 
 Nmap scan report for 175.45.177.218
+Not shown: 53771 closed ports, 11763 filtered ports
+PORT   STATE SERVICE VERSION
+23/tcp open  telnet?
+OS fingerprint not ideal because: Didn't receive UDP response. Please try again with -sSU
+No OS matches for host
 
 Nmap scan report for 175.45.177.219
 
@@ -1906,14 +1769,6 @@
 Nmap scan report for 175.45.179.16
 
 Nmap scan report for 175.45.179.17
-Not shown: 65534 filtered ports
-PORT      STATE SERVICE VERSION
-11000/tcp open  irisa?
-Warning: OSScan results may be unreliable because we could not find at least 1 open and 1 closed port
-Device type: WAP|switch
-Running (JUST GUESSING): D-Link embedded (96%), TRENDnet embedded (96%), HP embedded (94%)
-Aggressive OS guesses: D-Link DWL-624+ or DWL-2000AP, or TRENDnet TEW-432BRP WAP (96%), HP 4000M ProCurve switch (J4121A) (94%)
-No exact OS matches for host (test conditions non-ideal).
 
 Nmap scan report for 175.45.179.18
 
@@ -2007,409 +1862,3 @@
 
 Nmap scan report for 175.45.179.63
 
-Nmap scan report for 175.45.179.64
-
-Nmap scan report for 175.45.179.65
-
-Nmap scan report for ns1.star.edu.kp (175.45.179.66)
-
-Nmap scan report for email.kp.col.cn (175.45.179.67)
-Not shown: 65533 filtered ports
-PORT    STATE SERVICE VERSION
-25/tcp  open  smtp    Cisco PIX sanitized smtpd
-110/tcp open  pop3    Cyrus pop3d 2.3.7-Invoca-RPM-2.3.7-7.el5
-Warning: OSScan results may be unreliable because we could not find at least 1 open and 1 closed port
-Device type: switch|WAP
-Running (JUST GUESSING): HP embedded (96%), D-Link embedded (94%), TRENDnet embedded (94%)
-Aggressive OS guesses: HP 4000M ProCurve switch (J4121A) (96%), D-Link DWL-624+ or DWL-2000AP, or TRENDnet TEW-432BRP WAP (94%)
-No exact OS matches for host (test conditions non-ideal).
-Service Info: Device: firewall
-
-Nmap scan report for 175.45.179.68
-
-Nmap scan report for mail.star.edu.kp (175.45.179.69)
-Not shown: 65533 filtered ports
-PORT    STATE SERVICE VERSION
-25/tcp  open  smtp    Cisco PIX sanitized smtpd
-110/tcp open  pop3    Cyrus pop3d 2.3.7-Invoca-RPM-2.3.7-7.el5
-Warning: OSScan results may be unreliable because we could not find at least 1 open and 1 closed port
-Device type: switch|WAP
-Running (JUST GUESSING): HP embedded (96%), D-Link embedded (94%), TRENDnet embedded (94%)
-Aggressive OS guesses: HP 4000M ProCurve switch (J4121A) (96%), D-Link DWL-624+ or DWL-2000AP, or TRENDnet TEW-432BRP WAP (94%)
-No exact OS matches for host (test conditions non-ideal).
-Service Info: Device: firewall
-
-Nmap scan report for 175.45.179.70
-
-Nmap scan report for 175.45.179.71
-
-Nmap scan report for 175.45.179.72
-
-Nmap scan report for 175.45.179.73
-
-Nmap scan report for 175.45.179.74
-
-Nmap scan report for 175.45.179.75
-
-Nmap scan report for 175.45.179.76
-
-Nmap scan report for 175.45.179.77
-
-Nmap scan report for 175.45.179.78
-
-Nmap scan report for 175.45.179.79
-
-Nmap scan report for 175.45.179.80
-
-Nmap scan report for 175.45.179.81
-
-Nmap scan report for 175.45.179.82
-
-Nmap scan report for 175.45.179.83
-
-Nmap scan report for 175.45.179.84
-
-Nmap scan report for 175.45.179.85
-
-Nmap scan report for 175.45.179.86
-
-Nmap scan report for 175.45.179.87
-
-Nmap scan report for 175.45.179.88
-
-Nmap scan report for 175.45.179.89
-
-Nmap scan report for 175.45.179.90
-
-Nmap scan report for 175.45.179.91
-
-Nmap scan report for 175.45.179.92
-
-Nmap scan report for 175.45.179.93
-
-Nmap scan report for 175.45.179.94
-
-Nmap scan report for 175.45.179.95
-
-Nmap scan report for 175.45.179.96
-
-Nmap scan report for 175.45.179.97
-
-Nmap scan report for 175.45.179.98
-
-Nmap scan report for 175.45.179.99
-
-Nmap scan report for 175.45.179.100
-
-Nmap scan report for 175.45.179.101
-
-Nmap scan report for 175.45.179.102
-
-Nmap scan report for 175.45.179.103
-
-Nmap scan report for 175.45.179.104
-
-Nmap scan report for 175.45.179.105
-
-Nmap scan report for 175.45.179.106
-
-Nmap scan report for 175.45.179.107
-
-Nmap scan report for 175.45.179.108
-
-Nmap scan report for 175.45.179.109
-
-Nmap scan report for 175.45.179.110
-
-Nmap scan report for 175.45.179.111
-
-Nmap scan report for 175.45.179.112
-
-Nmap scan report for 175.45.179.113
-
-Nmap scan report for 175.45.179.114
-
-Nmap scan report for 175.45.179.115
-
-Nmap scan report for 175.45.179.116
-
-Nmap scan report for 175.45.179.117
-
-Nmap scan report for 175.45.179.118
-
-Nmap scan report for 175.45.179.119
-
-Nmap scan report for 175.45.179.120
-
-Nmap scan report for 175.45.179.121
-
-Nmap scan report for 175.45.179.122
-
-Nmap scan report for 175.45.179.123
-
-Nmap scan report for 175.45.179.124
-
-Nmap scan report for 175.45.179.125
-
-Nmap scan report for 175.45.179.126
-
-Nmap scan report for 175.45.179.127
-
-Nmap scan report for 175.45.179.128
-
-Nmap scan report for 175.45.179.129
-
-Nmap scan report for 175.45.179.130
-
-Nmap scan report for 175.45.179.131
-
-Nmap scan report for 175.45.179.132
-
-Nmap scan report for 175.45.179.133
-
-Nmap scan report for 175.45.179.134
-
-Nmap scan report for 175.45.179.135
-
-Nmap scan report for 175.45.179.136
-
-Nmap scan report for 175.45.179.137
-
-Nmap scan report for 175.45.179.138
-
-Nmap scan report for 175.45.179.139
-
-Nmap scan report for 175.45.179.140
-
-Nmap scan report for 175.45.179.141
-
-Nmap scan report for 175.45.179.142
-
-Nmap scan report for 175.45.179.143
-
-Nmap scan report for 175.45.179.144
-
-Nmap scan report for 175.45.179.145
-
-Nmap scan report for 175.45.179.146
-
-Nmap scan report for 175.45.179.147
-
-Nmap scan report for 175.45.179.148
-
-Nmap scan report for 175.45.179.149
-
-Nmap scan report for 175.45.179.150
-
-Nmap scan report for 175.45.179.151
-
-Nmap scan report for 175.45.179.152
-
-Nmap scan report for 175.45.179.153
-
-Nmap scan report for 175.45.179.154
-
-Nmap scan report for 175.45.179.155
-
-Nmap scan report for 175.45.179.156
-
-Nmap scan report for 175.45.179.157
-
-Nmap scan report for 175.45.179.158
-
-Nmap scan report for 175.45.179.159
-
-Nmap scan report for 175.45.179.160
-
-Nmap scan report for 175.45.179.161
-
-Nmap scan report for 175.45.179.162
-
-Nmap scan report for 175.45.179.163
-
-Nmap scan report for 175.45.179.164
-
-Nmap scan report for 175.45.179.165
-
-Nmap scan report for 175.45.179.166
-
-Nmap scan report for 175.45.179.167
-
-Nmap scan report for 175.45.179.168
-
-Nmap scan report for 175.45.179.169
-
-Nmap scan report for 175.45.179.170
-
-Nmap scan report for 175.45.179.171
-
-Nmap scan report for 175.45.179.172
-
-Nmap scan report for 175.45.179.173
-
-Nmap scan report for 175.45.179.174
-
-Nmap scan report for 175.45.179.175
-
-Nmap scan report for 175.45.179.176
-
-Nmap scan report for 175.45.179.177
-
-Nmap scan report for 175.45.179.178
-
-Nmap scan report for 175.45.179.179
-
-Nmap scan report for 175.45.179.180
-
-Nmap scan report for 175.45.179.181
-
-Nmap scan report for 175.45.179.182
-
-Nmap scan report for 175.45.179.183
-
-Nmap scan report for 175.45.179.184
-
-Nmap scan report for 175.45.179.185
-
-Nmap scan report for 175.45.179.186
-
-Nmap scan report for 175.45.179.187
-
-Nmap scan report for 175.45.179.188
-
-Nmap scan report for 175.45.179.189
-
-Nmap scan report for 175.45.179.190
-
-Nmap scan report for 175.45.179.191
-
-Nmap scan report for 175.45.179.192
-
-Nmap scan report for 175.45.179.193
-
-Nmap scan report for 175.45.179.194
-
-Nmap scan report for 175.45.179.195
-
-Nmap scan report for 175.45.179.196
-
-Nmap scan report for 175.45.179.197
-
-Nmap scan report for 175.45.179.198
-
-Nmap scan report for 175.45.179.199
-
-Nmap scan report for 175.45.179.200
-
-Nmap scan report for 175.45.179.201
-
-Nmap scan report for 175.45.179.202
-
-Nmap scan report for 175.45.179.203
-
-Nmap scan report for 175.45.179.204
-
-Nmap scan report for 175.45.179.205
-
-Nmap scan report for 175.45.179.206
-
-Nmap scan report for 175.45.179.207
-
-Nmap scan report for 175.45.179.208
-
-Nmap scan report for 175.45.179.209
-
-Nmap scan report for 175.45.179.210
-
-Nmap scan report for 175.45.179.211
-
-Nmap scan report for 175.45.179.212
-
-Nmap scan report for 175.45.179.213
-
-Nmap scan report for 175.45.179.214
-
-Nmap scan report for 175.45.179.215
-
-Nmap scan report for 175.45.179.216
-
-Nmap scan report for 175.45.179.217
-
-Nmap scan report for 175.45.179.218
-
-Nmap scan report for 175.45.179.219
-
-Nmap scan report for 175.45.179.220
-
-Nmap scan report for 175.45.179.221
-
-Nmap scan report for 175.45.179.222
-
-Nmap scan report for 175.45.179.223
-
-Nmap scan report for 175.45.179.224
-
-Nmap scan report for 175.45.179.225
-
-Nmap scan report for 175.45.179.226
-
-Nmap scan report for 175.45.179.227
-
-Nmap scan report for 175.45.179.228
-
-Nmap scan report for 175.45.179.229
-
-Nmap scan report for 175.45.179.230
-
-Nmap scan report for 175.45.179.231
-
-Nmap scan report for 175.45.179.232
-
-Nmap scan report for 175.45.179.233
-
-Nmap scan report for 175.45.179.234
-
-Nmap scan report for 175.45.179.235
-
-Nmap scan report for 175.45.179.236
-
-Nmap scan report for 175.45.179.237
-
-Nmap scan report for 175.45.179.238
-
-Nmap scan report for 175.45.179.239
-
-Nmap scan report for 175.45.179.240
-
-Nmap scan report for 175.45.179.241
-
-Nmap scan report for 175.45.179.242
-
-Nmap scan report for 175.45.179.243
-
-Nmap scan report for 175.45.179.244
-
-Nmap scan report for 175.45.179.245
-
-Nmap scan report for 175.45.179.246
-
-Nmap scan report for 175.45.179.247
-
-Nmap scan report for 175.45.179.248
-
-Nmap scan report for 175.45.179.249
-
-Nmap scan report for 175.45.179.250
-
-Nmap scan report for 175.45.179.251
-
-Nmap scan report for 175.45.179.252
-
-Nmap scan report for 175.45.179.253
-
-Nmap scan report for 175.45.179.254
-
-Nmap scan report for 175.45.179.255
-
-OS and Service detection performed. Please report any incorrect results at http://nmap.org/submit/ .
-Nmap done: 1024 IP addresses (1024 hosts up) scanned in 2097392.43 seconds
